app/templates/_keystone.js

```diff
@@ -48,7 +48,7 @@ keystone.init({
 	'session': true,
 	'auth': true,
 	'user model': '<%= userModel %>',
-	'cookie secret': '<%= cookieSecret %>'
+	'cookie secret': process.env.COOKIE_SECRET
```

Comment:
We can actually remove this line entirely, the setting already defaults to `process.env.COOKIE_SECRET` if provided.